data/coalition/coalition news.txt

```diff
@@ -0,0 +1,271 @@
+			`"Never been around an angry longcow, have you? Word of advice, avoid their sides you should."`
+			`"Stopped using vaccines on longcows, long ago we did. Remember which leg to apply them on, we never could."`
+			`"Wish understand my jokes, my beauties could. Many kicks out of them, they would get."`
+			`"Fresh steak here! First leg steak, second leg steak... a discount for tenth leg steak there is!"`
+			"You see a rancher walk out of a shop with a trolley full of bells. You wonder if they are for multiple longcows, or just a particularly long one."
+
+news "kimek farmer"
+	location
+		attributes "kimek" "farming"
+	name
+		word
+			"Kimek farmer"
+	message
+		word
+			`"When starting out on this business, I was, lost in the farm I got. Tricky to tell where you are, it is, when green is all you see."`
+			`"A Heliarch agent, I once was. Too stressful, it was for me, so this job I took. Messages from colleagues, I still get, but if see the fruits I plant here, they could, definitely join me, they would."`
+			`"Some fresh produce would you like, Captain? If purchase these, you do, teach you a family recipe I could. I'm sure that love it, a human would. Probably."`
+			"After noticing you looking at the produce, the farmer asks if you would like some free samples, before grabbing an entire box with various fruits and vegetables and handing it to you."
+			"Dozens of farmers in different stands call out to you as you walk by the spaceport, asking you to take a look at the special prices of today's "food expo." You later look into it, and turns out yesterday there also was an expo, as well as the day before that. Tomorrow's starts the moment today's ends."
+
+news "heliarch agent"
+	location
+		government "Coalition" "Heliarch"
+	name
+		word
+			"Heliarch agent"
+	message
+		word
+			`"Protect and serve."`
+			`"If any assistance you need, at your disposal I am."`
+			`"If any trouble you find, report it to us you must."`
+			`"Saryd, Kimek, Arach, human, no matter what species, to protect all of our citizens, committed we are."`
+			`"Enjoy my post well enough, I do, but still to get a break during the Coalition games to watch them, I am."`
+			`"Part of a Punisher crew, I am. Land, refuel, rest, and back to working on that magnificent ship, I am!"`
+			`"Offered the position of elder, I actually have been. Only, not interested in that governing activity, I am."`
+			`"Apologies, but make haste I must. Gone missing some garrison guns have, and back to cleaning Stalker engines I will be, if find them I don't!"`
+			`"Difficult sometimes it is, to be a mediator. If careful with my words I am not, a Saryd might think an Arach talking about longcows has insulted their mother."`
+			"An agent is trying to calm down a small crowd outside a local bar. After some minutes, the more agitated drunkards are convinced to accompany the agent to a nearby Heliarch office."
```

Comment:
To the best of my understanding, these messages will appear on the three rings. Considering that, it feels weird to me that there would be bars full of unruly drunkards in places dedicated almost solely to governance and research. Perhaps have that message appear on normal Coalition planets only, and not the three rings.

Reply:
Ringworlds are colossal structures, over a trillion Quarg can live in a complete one. All of the Coalition, 280 billion people, cannot hope to fill that, much less three (admittedly two being incomplete, but still), so the way I see it, while the Heliarchs are most prominent in the rings, and their primary functions are to serve as hubs for the Heliarch government, military and research teams, there is no reason to discourage civilians from settling there or having small businesses (which would also serve as amenities for the troops) there. Granted, many areas are restricted to non-Heliarchs, but I don't think they would stop a bar from being established near where civilian captains would go to either conduct business or relax.

Reply:
That makes sense.